tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Actions/Test_ImposeBoundaryConditions.cpp

```diff
@@ -270,30 +248,16 @@ struct Metavariables {
   enum class Phase { Initialization, Testing, Exit };
 };
 
-template <typename Component>
-using compute_items = typename Component::compute_tags;
-
 template <bool HasConservativeAndPrimitiveVars>
 void run_test() {
   using metavariables = Metavariables<HasConservativeAndPrimitiveVars>;
   using flux_comm_types = typename component<metavariables>::flux_comm_types;
   using my_component = component<metavariables>;
-  using simple_tags = typename my_component::simple_tags;
-  using compute_tags = typename my_component::compute_tags;
   const Mesh<2> mesh{3, Spectral::Basis::Legendre,
                      Spectral::Quadrature::GaussLobatto};
 
-  //      xi      Block       +- xi
-  //      |     0   |   1     |
-  // eta -+ +-------+-+-+---+ eta
-  //        |       |X| |   |
-  //        |       +-+-+   |
-  //        |       | | |   |
-  //        +-------+-+-+---+
-  // We run the actions on the indicated element.
```

Comment:
Why are you deleting this? I found this helpful to understand the element IDs that are constructed below.

Reply:
Because it doesn't match the neighbor configuration used in the test.  The test element has two internal and two external boundaries, but the diagram shows three and one.  (Admittedly, having an external boundary in the +xi direction is kind of questionable.)

Reply:
Can you just update the diagram then? It‘s useful to understand which boundaries the test is working with

Reply:
The test places an external boundary in the middle of a block.  I don't have a good idea for how to draw that.  Do you have a suggestion?

Reply:
Can you make the `element_id`s and external boundaries consistent fairly easily, and add a comment and/or a diagram that shows the setup? I leave it up to you if you want to do that or if you think the current setup is fine.

Reply:
The current setup is not ideal, since it isn't valid for an evolution and I don't see any other reason we'd want to support it.  Unfortunately, I don't know what this test is doing, so I don't know if the current setup was chosen (or rather kept from the other tests this is based on) in order to test something specific.  @osheamonn?

Reply:
Let's move this to an issue and merge this PR :)

Reply:
https://github.com/sxs-collaboration/spectre/issues/1679